api/db-cdr/generate-cdr/cb-criteria-parallel/debug/make-cb-criteria-00-main-tables.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# This script does the following
+#   populate all cb_criteria table *in order*
+#   populate cb_criteria_* abd cb_survey_* tables
+echo "-----------------------------------------------------------"
+echo "Started at "`date`
+echo "PID "$$
+
+set -e
+# vars are purposely hard-coded for iterative testing
+export BQ_PROJECT=$1       # project - 'all-of-us-ehr-dev'
+export BQ_DATASET=$2       # dataset - 'BillDummyMult'
+
+run_in_parallel=$1
```

Comment:
This is no longer referenced in this script? Should it be removed?

Reply:
done